src/directions/types.ts

```diff
@@ -284,17 +284,20 @@ export type GeoJSONGeometry = { coordinates: [number, number][] };
 export type PolylineGeometry = string;
 
 export interface Route {
+  [P: string]: unknown;
```

Comment:
Why do you use unknown here?

Reply:
@wipfli 'cause I don't want to type the whole response since not all the fields are used by the lib. Having index signature allows the user to cast types when needed. Something like `e.data?.routes[0].distance as number` e.g. I could use `any` here, but that's worse than `unknown` because when it's `unknown` the user is forced to assert the appropriate type manually, and `any` on the other hand introduces potential runtime errors.